tests/test_identifiers.py

```diff
@@ -234,12 +234,11 @@ def test_create_identifiers_not_exists(self):
             res.json['doi'],
             self.node.get_identifier_value('doi')
         )
-        assert_equal(
-            res.json['ark'],
-            self.node.get_identifier_value('ark')
-        )
-        assert_equal(res.status_code, 201)
 
+        with assert_raises(KeyError):
```

Comment:
This would be slightly a clearer assert if it was something like `assert_not_in(res.json.keys(), 'ark')`, as you don't really need to save the `ark` variable